lectures/unpleasant.md

```diff
@@ -36,18 +36,20 @@ by printing money at times $t \geq T$.
 
 These outcomes are the essential finding of Sargent and Wallace's **unpleasant monetarist arithmetic** {cite}`sargent1981`.
 
-**Reader's Guide:** Please read our lecture on Money Supplies and Price levels before diving into this lecture.
+```{tip}
+Please read our lecture on {doc}`money supplies and price levels<money_inflation>` before diving into this lecture.
+```
 
 That lecture  described  supplies and demands for money that appear in lecture.
 
 It also   characterized the steady state equilibrium from which we work backwards in this lecture. 
 
-In addition to learning about ''unpleasant monetarist arithmetic", in this lecture we'll learn how to implement a **fixed point** algorithm for computing an initial price level.
+In addition to learning about **unpleasant monetarist arithmetic**, in this lecture we'll learn how to implement a *fixed point* algorithm for computing an initial price level.
```

Comment:
`**` is for definitions -- I suggest reverting to simple quotes `"`.

For fixed point, I suggest converting it to a link to https://en.wikipedia.org/wiki/Fixed_point_(mathematics)